crates/iota-benchmark/src/lib.rs

```diff
@@ -625,7 +625,9 @@ impl ValidatorProxy for LocalValidatorAggregatorProxy {
     async fn get_validators(&self) -> Result<Vec<IotaAddress>, anyhow::Error> {
         let active_validators = match self.get_latest_system_state_object().await? {
             IotaSystemStateSummary::V1(v1) => v1.active_validators,
-            IotaSystemStateSummary::V2(v2) => v2.active_validators,
+            IotaSystemStateSummary::V2(v2) => {
+                v2.iter_committee_members().cloned().collect::<Vec<_>>()
```

Comment:
should we add a func that returns this vector?

Reply:
You could even add another one for the IotaSystemState enum which does this match and returns the active_validators independent of the version

Reply:
Yes using the `Either` to enhance it. (Credits to @semenov-vladyslav )